### ISSUE TITLE
Error in the social card plugin

### ISSUE THREAD

**Issue by faridshamsutdinov** [squidfunk/mkdocs-material#5505]
### Context

After a recent update (I suspect this [one](https://github.com/squidfunk/mkdocs-material-insiders/commit/e3c253fd455dd7e7afcf9f2b3dcfec24ee816496)) an error came up during build that swore it was the wrong path to the logo in .SVG format.

### Bug description

```
INFO     -  DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for
            alternative uses
              File "/usr/lib/python3/dist-packages/apport/report.py", line 13, in <module>
                import fnmatch, glob, traceback, errno, sys, atexit, imp, stat
              File "/usr/lib/python3.8/imp.py", line 31, in <module>
                warnings.warn("the imp module is deprecated in favour of importlib; "
Traceback (most recent call last):
  File "/home/farid/.local/bin/mkdocs", line 8, in <module>
    sys.exit(cli())
  File "/usr/lib/python3/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/mkdocs/__main__.py", line 250, in build_command
    build.build(cfg, dirty=not clean)
  File "/home/farid/.local/lib/python3.8/site-packages/mkdocs/commands/build.py", line 329, in build
    _build_page(file.page, config, doc_files, nav, env, dirty)
  File "/home/farid/.local/lib/python3.8/site-packages/mkdocs/commands/build.py", line 234, in _build_page
    output = config.plugins.run_event('post_page', output, page=page, config=config)
  File "/home/farid/.local/lib/python3.8/site-packages/mkdocs/plugins.py", line 520, in run_event
    result = method(item, **kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 151, in on_post_page
    raise future.exception()
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 284, in _generate
    self.card_layer_jobs[h].result(),
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 444, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 316, in _render
    image = self._render_background(layer, image)
  File "/home/farid/.local/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 332, in _render_background
    with open(background["image"], "rb") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'assets/gru/logo.svg'
```

### Related links

- [Possible error here](https://github.com/squidfunk/mkdocs-material-insiders/commit/e3c253fd455dd7e7afcf9f2b3dcfec24ee816496)

### Reproduction

No file

### Steps to reproduce

Connect plugin `social` and in the theme specify the logo in the format `.svg

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by squidfunk** [comment]
Found the problem – the logo must be located in the `docs_dir`, but the cards were assuming it's referenced from the project root. Fixed it in [`bf6efe45f`](https://github.com/squidfunk/mkdocs-material-insiders/commit/bf6efe45fd35b4c522b67e306ae45f28652e6180).

**Comment by squidfunk** [comment]
Released as part of [9.1.12+insiders-4.33.2](https://squidfunk.github.io/mkdocs-material/insiders/changelog/#4.33.2).
